packages/core/test/ContentSerdesTest.ts

```diff
@@ -742,11 +766,23 @@ class SerdesOctetTests {
     }
 
     @test "value to OctetStream should throw"() {
-        // @ts-ignore new dataschema types are not yet supported in the td type definitions
-        expect(() => ContentSerdes.valueToContent(2345, { type: "int8" }, "application/octet-stream")).to.throw(
+        expect(() => ContentSerdes.valueToContent(256, { type: "uint8" }, "application/octet-stream")).to.throw(
+            Error,
+            "Integer overflow when representing unsigned 256 in 8 bit(s)"
```

Comment:
```suggestion
            "Integer overflow when representing 256 as uint8"
```

The error messages are confusing, same for the others. Or is it just me?

Reply:
It somewhat is, would you prefer something along:
```
input requires 9 bits; 8 bits were specified
```
or do you have a different proposal?

Reply:
I sketched above what I think is more clear. The same applies for the other messages...

"Integer overflow when representing unsigned 256 in 8 bit(s)"
-->
"Integer overflow when representing 256 as uint8"

Reply:
Sorry, somehow I completely missed the suggestion :smile:

The suggest can't be applied to the error message returned by the [test in lines 801 - 807](https://github.com/derwehr/node-wot/blob/1aa6559a92d282b9c3cc03cad28c34de0aadbb6f/packages/core/test/ContentSerdesTest.ts#L807) though, and it would require to add a `type` argument to the `valueToInteger` method.

Therefore, I'd suggest
```
Integer overflow when representing -2345 as a signed integer using 10 bit(s)
```

Reply:
That makes sense